UkolPlanovac/Program.cs

```diff
@@ -15,18 +15,12 @@ static void Main(string[] args)
 
                 if (input.Contains("EVENT;"))
                 {
-                    bool isEventValid;
-                    string name;
-                    DateTime date;
-                    Event newEvent;
-                    
-                    (isEventValid, name, date) = Event.CheckInput(input);
+                    (bool isEventValid, string name, DateTime date) = Event.ParseEvent(input);
```

Comment:
Soucasti dekonstrukce tuple muzeme rovnou deklarovat nove promenne 🙂 Alternativne to muzeme dale zjednodussit pomoci `var`: 

```suggestion
                    var (isEventValid, name, date) = Event.ParseEvent(input);
```